iFixit/checkout-ui-extensions
# Implement Dropshipping Notice

## Overview

We want to apply a notice on the shipping options screen if the checkout has dropshipped items. If the entire order is dropshipped, we give one notice, and if only some of the order is dropshipped we show another notice.

## QA

Make sure the above is true. I would use [this](https://admin.shopify.com/store/ifixit-checkout-dev/products/9003763269917) or [this](https://admin.shopify.com/store/ifixit-checkout-dev/products/9003762975005) as the dropshipped products, as i have configured their metafields correctly already. Also if no products are dropshipped in the order than no notice should appear.
Example of a possible notice
![image_480](https://github.com/iFixit/checkout-ui-extensions/assets/109113415/8b539cc0-7f29-49ff-828b-0c1519a8284f)

Closes: https://github.com/iFixit/checkout-ui-extensions/issues/30
